torch/csrc/jit/passes/normalize_ops.h

```diff
@@ -12,5 +12,7 @@ namespace jit {
 // Currently only handles normalization of op aliases.
 TORCH_API void NormalizeOps(const std::shared_ptr<Graph>& graph);
 
+extern const std::unordered_map<Symbol, Symbol> alias_map;
```

Comment:
nit: I would have some slight preference to making this an AP to get the map instead of exposing the map directly. i think the main reason is to not clutter the namespace, & that seems to be how we do it in some other files https://github.com/pytorch/pytorch/blob/cebe87fe3a66a74b6fdbb260bdfadb453444a151/torch/csrc/jit/frontend/sugared_value.cpp#L34. 

but this is a nit / personal preference, it's up to you.

Reply:
I'd prefer to keep my changes consistent with the rest of the code base. Thanks for pointing this out. I've changed it.